rasa/server.py

```diff
@@ -771,6 +771,10 @@ def _get_events_from_request_body(request: Request) -> List[Event]:
             stories_path = os.path.join(temp_dir, "stories.md")
             rasa.utils.io.write_text_file(rjs["stories"], stories_path)
 
+        if "responses" in rjs:
+            responses_path = os.path.join(temp_dir, "responses.md")
+            rasa.utils.io.write_text_file(rjs["responses"], responses_path)
+
```

Comment:
it would be great to add a unit test for successful training with responses in `test_server.py`